.eslintrc.json

```diff
@@ -10,6 +10,14 @@
         "arrow-spacing": ["error", { "before": true, "after": true }],
         "brace-style": ["error", "1tbs", { "allowSingleLine": true }],
         "constructor-super": "error",
+        "indent": ["error", "tab", {
```

Comment:
CallExpression indenting isn't on by default. Setting it as I have done forces arguments to function calls on subsequent lines to be indented one tab.

```js
QUnit.on(
    "event", // <-- poor man's tab
    doSomething);
```

MemberExpression does a similar thing with property accesses and would also be off by default:

```js
return myString
    .replace("foo", "bar")
    .replace("baz", "quux");
```

SwitchCase enforces this style of case statements (it's set same as default, but since it was called out in the style guide I thought it was good to include in case the default changes upstream):

```js
switch (x) {
case "foo":
    return true;
case "bar":
    return false;
```

And outerIIFEBody is for full file closures... at least, those that are actually pure IIFEs. Setting it to 1 means basically all function declarations/expressions/IIFEs require 1 tab indent of statements in the body.